Project.toml

```diff
@@ -16,6 +16,10 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
+julia = "^1.6.0"
 LDLFactorizations = "0.8"
 PrettyTables = "1"
 StatsBase = "0.33"
```

Comment:
What are the two above needed for?

Reply:
I use `StatsBase` for the `sample()` function, which is a `rand()` without repetetion.
It is used to define random partitioned structures.

`PrettyTables` is currently not used.

Reply:
Ok. What's the use of defining random structures? If it's for the unit tests, then StatsBase should only be an "extra" dependency (to be used in the unit tests).

Reply:
It is used in the definition of the partitioned structures itself.
These random structures are mainly defined for tests.

Reply:
Ok, I think we also need to open an issue to move StatsBase to the "Extra" dependencies in the future.